app/models/codelist.rb

```diff
@@ -38,6 +38,22 @@ def initialize_codelists
     end
   end
 
+  def hash_of_coded_names
+    coded_values = {}
+    list.each do |code|
+      coded_values[ActiveSupport::Inflector.parameterize(code["name"], separator: "_").to_sym] = code["code"]
```

Comment:
I would avoid using `to_sym` here and leave these values as strings -- they're data, not names of things internal to the program like method names or column names. This also makes this consistent with the type of `hash_of_named_codes` which has string keys.